src/platform/packages/shared/kbn-esql-validation-autocomplete/scripts/generate_function_definitions.ts

```diff
@@ -25,6 +30,52 @@ const aliasTable: Record<string, string[]> = {
 };
 const aliases = new Set(Object.values(aliasTable).flat());
 
+const groupingTypeTable: Array<
```

Comment:
Is this just for bucket signatures? If so, perhaps we rename it to `bucketParameterTypes` or something

Reply:
Sure, I renamed it here https://github.com/elastic/kibana/pull/210513/commits/839f606aa4c0755ccf7aa60ecd7d34a42adef40c